app/forms/user_piv_cac_setup_form.rb

```diff
@@ -0,0 +1,91 @@
+class UserPivCacSetupForm
+  extend Forwardable
+
+  State = Struct.new(:x509_dn_uuid, :x509_dn, :error, :success)
```

Comment:
What do you think about using ActiveModel validations instead, like in our other forms, for consistency?